### DISCUSSION TITLE
Linking to myNCI and Teams on the documentation site

### DISCUSSION THREAD

**Discussion by bennettcc** [NCIOCPL/ncids#1374] [NCIDS Launch]
Submitted by Anna during Doc site review: 

Should we be linking to myNCI and Teams on the documentation site for the NCIDS launch?

Example: https://designsystem-dev.cancer.gov/develop/about/

**Comment by blilianyu** [comment]
_Previously discussed in a requirements meeting._

We determined we would link to the NCI OCPL Web Design System Teams channel on the [About the NCI Design System page](https://designsystem-dev.cancer.gov/develop/about/) and chose against linking to MyNCI [[#1220 (https://github.com/NCIOCPL/ncids/issues/1220)]. 

For the related discussion topic, if there are specific MyNCI links we want to link to, sounds like we would need to align with Anna.

cc: @monika-jaeger

**Comment by laurelthrash** [comment]
Discussed during 04/23 UX meeting that this has been addressed for the time being and the discussion can be closed out.
